event-exporter/sinks/stackdriver/log_entry_factory.go

```diff
@@ -62,10 +66,14 @@ func (f *sdLogEntryFactory) FromEvent(event *api_v1.Event) *sd.LogEntry {
 		glog.Warningf("Failed to encode event %+v: %v", event, err)
 	}
 
+	resource := f.resourceFactory.resourceFromEvent(event)
```

Comment:
@igorpeshansky
Can you have a look on this? We will add label containing event's involved object for every log entry. This way we will not loose events for objects like 'replicaSet' 'deployment' etc

label name is: 'involved_object'